torch/csrc/jit/codegen/cuda/scheduler/transpose.cpp

```diff
@@ -59,9 +59,42 @@ class DomainMap : public pointwise_utils::DomainMap {
   }
 
   int getPosMappedTo(TensorView* tv, IterDomain* id) const {
+    // Find the root id mapped to `id`
+    const auto& root_dom = tv->getRootDomain();
```

Comment:
Should this not be on rfactor domain?

Reply:
I will revisit this later when we enable transpose+view